file/reader.go

```diff
@@ -60,12 +65,16 @@ func GetForKonnect(fileContent *Content, opt RenderConfig) (*utils.KongRawState,
 
 // Get process the fileContent and renders a RawState.
 // IDs of entities are matches based on currentState.
-func Get(fileContent *Content, opt RenderConfig, dumpConfig dump.Config) (*utils.KongRawState, error) {
+func Get(ctx context.Context, fileContent *Content, opt RenderConfig, dumpConfig dump.Config, wsClient *kong.Client) (
+	*utils.KongRawState, error,
```

Comment:
This is an ever growing list. Could you please add a Github issue to take care of this in future?
We need to replace all the parameters with a "GetOpts" struct and then pass that around.

Reply:
https://github.com/Kong/deck/issues/567